src/core/thread/mle_tlvs.hpp

```diff
@@ -444,7 +416,7 @@ class RouteTlv : public Tlv, public TlvInfo<Tlv::kRoute>
      * @retval FALSE  If the TLV does not appear to be well-formed.
      *
      */
-    bool IsValid(void) const { return GetLength() >= sizeof(mRouterIdSequence) + sizeof(mRouterIdMask); }
+    bool IsValid(void) const { return (GetLength() >= sizeof(mRouterIdSequence) + sizeof(mRouterIdMask)); }
```

Comment:
Just curious - is this change for readability?

Reply:
Oh no! This is under `LONG_ROUTE` config. I changed it by mistake.  Will change it back.